bitnami/kafka/README.md

```diff
@@ -146,61 +146,63 @@ The command removes all the Kubernetes components associated with the chart and
 | `auth.tlsEndpointIdentificationAlgorithm`  | DEPRECATED: use `auth.tls.endpointIdentificationAlgorithm` instead.                                                                                  | `https`                             |
 | `listeners`                                | The address(es) the socket server listens on. Auto-calculated it's set to an empty array                                                             | `[]`                                |
 | `advertisedListeners`                      | The address(es) (hostname:port) the broker will advertise to producers and consumers. Auto-calculated it's set to an empty array                     | `[]`                                |
-| `listenerSecurityProtocolMap`              | The protocol->listener mapping. Auto-calculated it's set to nil                                                                                      | `""`                                |
+| `listenerCProtocolMap`              | The protocol->listener mapping. Auto-calculated it's set to nil                                                                                      | `""`                                |
```

Comment:
Why did you change this? I mean, the parameter hasn't been renamed.

Reply:
yes you are right, it's my bad sorry.